Dockerfile

```diff
@@ -51,6 +32,34 @@ RUN mkdir /tmp/empty_project \
     && cargo +nightly-2022-08-08 add serde \
     && rm -rf /tmp/empty_project
 
+# `uv` is required for building c2rust-refactor
+RUN curl -LsSf https://astral.sh/uv/install.sh | sh
+ENV PATH="/root/.local/bin:${PATH}"
```

Comment:
`uv` is already installed in a later step.  Can you delete that now?  But move the `uv python install` up here, too.

Reply:
I removed the duplicate installation, but left `uv python install` where it is, since that seems related to the other discussion about what `uv`-related steps are actually needed to build c2rust.